modules/mailer.js

```diff
@@ -31,7 +32,7 @@ async function sendReminders(classInfo, clientEmails){
                 from: '"OG YOGA" ' + process.env.SENDER_EMAIL,
                 to: process.env.SENDER_EMAIL,
                 bcc: clientEmails,
-                subject: TODO,
+                subject: "TEST EMAIL",
```

Comment:
Let's change this to OGYoga: your class <class_name> starts in 15 minutes.

Reply:
Changed